coap-service/coap_service_api.h

```diff
@@ -204,41 +204,9 @@ int8_t coap_service_register_uri(int8_t service_id, const char *uri, uint8_t all
  * \param service_id       Id number of the current service.
  * \param *uri             Uri address.
  *
- * \return 0 for success / -1 for failure 
- */
-int8_t coap_service_unregister_uri(int8_t service_id, const char *uri);
-
-
-/**
- * \brief Register secure callback methods to CoAP server.
- *
- * Register application and informs CoAP services secure registery callback functions.
- *
- * \param service_id        Id number of the current service.
- * \param *start_ptr        Callback to inform security handling is started.
- * \param *security_done_cb Callback to inform security handling is done.
- *
- * \return -1               for failure 
- *          instance_id     For success (is used to identify registery)
- *
- */
-int8_t coap_service_register_uri_secure_cb_set(int8_t service_id, coap_service_security_start_cb *start_ptr, coap_service_security_done_cb *security_done_cb);
```

Comment:
this functionshould be back user might want to change the callbacks even after service is created and now it is impossible